src/Context/JsonContext.php

```diff
@@ -119,9 +128,15 @@ public function theJsonNodeShouldBeNull($node)
      */
     public function theJsonNodeShouldNotBeNull($node)
     {
-        $this->not(function () use ($node) {
```

Comment:
Why you don’t use the `not` wrapper here?

Reply:
You are right, I could have.

The only "issue" I see is that all those function need to mock or instantiate Mink instance, and I'm not really confident with that for now.

If I change back to the `not`, I can not easily test the function.

Reply:
You are right, I could have.

The only "issue" I see is that all those function need to mock or instantiate Mink instance, and I'm not really confident with that for now.

If I change back to the not, I can not easily test the function.